### PR TITLE
Fix/bugs in negated #12781

### PR DESCRIPTION
**Pull request by KushalSr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,11 +29,16 @@
...
+            "resolve"
+          ]
         }
       ],
       "no-use-before-define": "warn",
       "import/no-unresolved": "warn"
     }
+  },
+  "dependencies": {
+    "underscore": "^1.13.6"
```
Comment (StorytellerCZ) [inline package.json:42]: Adding this does not seem to be related to the issue.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (KushalSr) [reply package.json]: @StorytellerCZ I've fixed that. Please check once.

Thanks

**Comment by StorytellerCZ** [comment]
This has already been solved in #12781 and will be released in 2.14
